imqsauth/http.go

```diff
@@ -373,6 +375,55 @@ func httpHandlerCreateUser(central *ImqsCentral, w http.ResponseWriter, r *httpR
 	}
 }
 
+func httpHandlerCreateGroup(central *ImqsCentral, w http.ResponseWriter, r *httpRequest) {
+	groupname := strings.TrimSpace(r.http.URL.Query().Get("groupname"))
+	if groupname == "" {
+		authaus.HttpSendTxt(w, http.StatusNotAcceptable, "Group name may not be blank.")
+		return
+	}
+
+	if _, error := authaus.LoadOrCreateGroup(central.Central, groupname, true); error == nil {
+		central.Central.Log.Printf("New group added: '" + groupname + "'")
+		authaus.HttpSendTxt(w, http.StatusOK, "Group created ('"+groupname+"')")
+		return
+	} else {
+		central.Central.Log.Printf("Error creating group ('" + groupname + "') ")
```

Comment:
Use %v for printf, for example:
central.Central.Log.Printf("New group added: '%v', groupname)